app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>List of all users</h1>
+<div class='template'>
+  <% @users.each do |user| %>
+      <div class='user-container'>
+          <img src='<%= user.photo %>' alt='User Profile photo' class='user-img' />
+        <div class='user-div'>
+          <h2 class='username'>
+            <%= link_to user.name, user_url(user) %>
+          </h2>
+          <small class='posts_counter'>Number of posts: <%= user.posts_counter %></small>
+        </div>
+      </div>
+  <% end %>
+</div>
```

Comment:
[OPTIONAL] In scenarios like this it is a good practice to use rails partials. Kindly read up on it.

Reply:
Great advice. Much appreciated @fobadara